tests/IBTK/curl_01_2d.conforming.output

```diff
@@ -0,0 +1 @@
+|e|_oo = 0.195336
```

Comment:
Interestingly, this choice halves the error.

Reply:
Hmmm.